### PR TITLE
To text inputs

### PR DESCRIPTION
**Pull request by n-lina**

### CODE REVIEW

**Review by pmirski — APPROVED** [review]

File: frontend/app/models/award-store/award-store.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { Instance, SnapshotOut, types } from "mobx-state-tree"
+import { string } from "mobx-state-tree/dist/internal"
```
Comment (jeremywiens) [inline frontend/app/models/award-store/award-store.ts:2]: Could we remove this import

File: frontend/app/models/award-store/award-store.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import { withEnvironment } from "../extensions/with-environment";
+
+/**
+ * Model description here for TypeScript hints.
+ */
+export const AwardStoreModel = types
+  .model("AwardStore")
+  .props({
+    awards: types.optional(types.array(types.array(string)), [["No Awards Yet ...", "Keep completing goals to get one soon!"]]),
```
Comment (jeremywiens) [inline frontend/app/models/award-store/award-store.ts:11]: Here we should have `types.string`

File: frontend/app/models/award-store/award-store.ts (hunk 1 above)
Comment (n-lina) [reply frontend/app/models/award-store/award-store.ts]: yes removed!

File: frontend/app/models/award-store/award-store.ts (hunk 2 above)
Comment (n-lina) [reply frontend/app/models/award-store/award-store.ts]: fixed

File: frontend/app/models/award-store/award-store.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  .actions(self => ({
+    setAwards: (AwardsList) => {
+      if (AwardsList) {
+        __DEV__ && console.log("Setting Awards list " + AwardsList.toString());
+        self.awards = AwardsList;
+      } else { __DEV__ && console.log("Unsetting Awards list"); }
+    },
+  })).actions( self => ({
+    getAllAwards: () => {
+      return self.environment.api.getAllAwards().then(res => {
```
Comment (jeremywiens) [inline frontend/app/models/award-store/award-store.ts:24]: I think we decided that the awards would just be a frontend thing for now, correct?

Comment (n-lina) [reply frontend/app/models/award-store/award-store.ts:24]: yes

**Pull request merged**